app/views/shared/_footer.html.haml

```diff
@@ -140,8 +140,6 @@
           = t '.footer_legal_text_html', {content_license: link_to('CC BY-SA 3.0', 'https://creativecommons.org/licenses/by-sa/3.0/'), code_license: link_to('AGPL 3', 'https://tldrlegal.com/license/gnu-affero-general-public-license-v3-(agpl-3.0)' )}
         %p.text-small
           %div
-            - cookies_policy_link = link_to( t( '.footer_data_cookies_policy' ), '', 'cookies-policy-modal' => true, 'cookies-banner' => !CookiesConsent.new(cookies, request.host).exists? && Spree::Config.cookies_consent_banner_toggle)
-            - privacy_policy_link = link_to( t( '.footer_data_privacy_policy' ), Spree::Config.privacy_policy_url, :target => '_blank' )
```

Comment:
I see you've deleted these two lines and added methods for these in the `FooterLinksHelper` module, but won't you still need to call those methods from here instead of removing them entirely?

Reply:
if I understand your doubt, yes, they are used in the lines below, right?